### ISSUE TITLE
Occlusion between different classes in multi modal GT-Sampling.

### ISSUE THREAD

**Issue by rkotimi** [dvlab-research/FocalsConv#11]
Hello @yukang2017 ! I notice that your FocalsConv has been supported by OpenPCDet. Congratulations!

After reading your code implementation of multi modal GT-Sampling, I think you only deal with the occlusion of objects belong to the same class on the image, which may case severe occlusion between different classes. Is it a potential bug? May it lead to suboptimal performance?

**Comment by yukang2017** [comment]
Hi! Multi-modal gt-sampling indeed will cause occlusions in images (as the image blew). It indeed leads suboptimal performance on some categories on nuScenes. You can refer to the Table S - 16 in the paper. 

I currently has no idea to solve this problem elegantly. We might reply on pre-trained instance segmentation models to copy paste mask instead of boxes to relieve the occlusion. But this inevitably complicates the pipeline.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/24690867/180348304-3c6173e8-825a-4757-988f-c8a71d61073e.png">

![](https://user-images.githubusercontent.com/24690867/168419982-dedee243-d77a-4f3f-94f1-310aa7fbc9b5.png)
